src/chigame/games/views.py

```diff
@@ -58,6 +59,35 @@ class ViewLobbyDetails(DetailView):
     context_object_name = "lobby_detail"
 
 
+class LobbyUpdateView(UpdateView):
+    model = Lobby
+    form_class = LobbyForm
+    template_name = "games/lobby_form.html"
+
+    def get_success_url(self):
+        return reverse_lazy("lobby-details", kwargs={"pk": self.object.pk})
+
+    def dispatch(self, request, *args, **kwargs):
+        # get the lobby object
+        self.object = self.get_object()
+        # check if the user making the request is the "host" of the lobby
+        if request.user != self.object.created_by:
```

Comment:
I don't think it was in your user stories, but it would make sense to also allow staff to edit or delete lobbies (in the case that lobbies have expired, modifications necessary, etc.)

Reply:
Thank you for the suggestion. Adjusted!